tests/misc_env/install_prereq.py

```diff
@@ -115,12 +129,26 @@ def setup_subscription_manager(ceph, timeout=1800):
     )
     while True:
         try:
-            ceph.exec_command(
-                cmd="sudo subscription-manager --force register  "
-                "--serverurl=subscription.rhsm.stage.redhat.com:443/subscription  "
-                "--baseurl=https://cdn.redhat.com --username=rhcsuser --password=rhcsuser",
-                timeout=720,
-            )
+            command = "sudo subscription-manager --force register --username={username} --password={password} "
+            if is_production:
+                credential = get_cephci_config().get("cdn_credentials")
+                command = command.format(
+                    username=credential.get("username"),
+                    password=credential.get("password"),
+                )
+
+            else:
+                credential = get_cephci_config().get("stage_credentials")
```

Comment:
I don't think we should use `.get()` unless it is truly an optional setting.

If a user lacks `cdn_credentials` or `stage_credentials` in `~/.cephci.yaml`, and we call into `setup_subscription_manager()`, what is the expected behavior?

Reply:
@ktdreyer .get has been avoided
If user miss credentials, newly added Attribute Error would catch and throw error